File: PokeIn/PokeIn_Free_v2.032/PokeIn_Free_v2.032/VB.NET MVC3 Razor/VB.NET MVC3/Models/SampleClass.vb
Imports PokeIn
Imports PokeIn.Comet

Public Class SampleClass 
    Implements IDisposable

    Private ClientID As String

    Public Sub New(ByVal clientId As String)
        Me.ClientID = clientId
    End Sub

    Public Overloads Sub Dispose() Implements IDisposable.Dispose
        'Do whatevet you want here, PokeIn will call this method when a user is disconnected
    End Sub

    Public Sub GetServerTime()
        Dim msg As String
        msg = JSON.Method("ServerTime", DateTime.Now)
        CometWorker.SendToClient(ClientID, msg)
    End Sub
End Class

File: PokeIn/PokeIn_Free_v2.032/PokeIn_Free_v2.032/MVC3 Razor CS/MVC3 Razor CS/Models/SampleClass.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using PokeIn;
using PokeIn.Comet;

namespace MVC3_Razor_CS.Models
{
    public class SampleClass:IDisposable
    {
        string _clientID;
        public SampleClass(string clientId)
        {
            _clientID = clientId;
        }

        public void Dispose()
        {
            //Do whatever you want here : PokeIn is going to call this method when user disconnectes
        }

        public void GetServerTime()
        {
            string msg = JSON.Method("ServerTime", DateTime.Now);
            CometWorker.SendToClient(_clientID, msg);
        }
    }
}